riscos_toolbox/gadgets/button.py

```diff
@@ -45,13 +45,16 @@ def validation(self, validation):
 
     @property
     def font(self):
-        return swi.swi('Toolbox_ObjectMiscOp','0III',
-            self.window.id, Button.GetFont)
+        raise RuntimeError("Get font for button is not implemented.")
 
     @font.setter
-    def fond(self, font):
-        font, width, height = font
-        swi.swi('Toolbox_ObjectMiscOp','0IIIsII',
+    def font(self, font):
+        if len(font) == 2:
```

Comment:
I think for this (and others, as well) you should be including a function comment to describe what parameters it takes.

This is true of many of these interfaces, and should be done anyhow, but here the interface is non-obvious. Personally, I'd have expected a font assignment to take a font name, not a tuple - that really feels like a method call to me.

ie, using...

```
button.font = "Homerton.Medium"
```

would feel natural.

Or maybe...

```
button.font = Font("Homerton.Medium", 12, 12)
```

But because this is a composite, you might just be better using a method to set the font, instead of trying to shoe-horn it into a single value.
But using...

```
button.font = ("Homerton.Medium", 12, 12)
```

feels a bit more magic.